snap/snapcraft.yaml

```diff
@@ -111,15 +111,15 @@ parts:
     plugin: nil
     source: https://git.launchpad.net/ubuntu/+source/aml
     source-type: git
-    source-branch: applied/ubuntu/mantic-devel
+    source-branch: applied/ubuntu/noble-devel
     source-depth: 1
 
   neatvnc:
     after: [ffmpeg]
     plugin: nil
     source: https://git.launchpad.net/ubuntu/+source/neatvnc
     source-type: git
-    source-branch: applied/ubuntu/mantic-devel
+    source-branch: applied/ubuntu/noble-devel
```

Comment:
I think we also need:

```yaml
build-packages:
  ...
  - libxxhash-dev

...
stage-packages:
  ...
  - libxxhash0
```

Reply:
`apt show libxxhash-dev` says it contains a static library so, maybe, there's no need for the shared library

Reply:
> `apt show libxxhash-dev` says it contains a static library so, maybe, there's no need for the shared library

Indeed, seems to work without.